### DISCUSSION TITLE
Aubout USB Host Controller of hobbytronics

### DISCUSSION THREAD

**Discussion by liqijian101** [luc-github/ESP3D#789] [General]
Did anyone make a [USB Host Controller of Hobbytronics](https://www.hobbytronics.co.uk/usb-host/usb-host-board-v24).

At the topic of  #424 , I found that we could not copy this module.
Because we don't have a bootloader for this module.
How do we download the program, using PICKIT2？

@mrv96

https://github.com/luc-github/ESP3D/issues/424#issuecomment-621151937

"""""
I checked the BOM. If you take the components from LCSC, and order non-assembled PCBs from JLCPCB, you’ll pay:
-22.62$ of components for each PCB
-2$ for 5 PCBs (if you panelize the PCB you can get 10 PCBs for 2$)

Shipping cost and Wemos D1 Mini excluded.
WARNING: In order to flash the bootloader on the PIC (only needed one time) it’s needed a PIC programmer (e.g. PicKit 3 or 4).
""""

**Comment by luc-github** [comment]
The fw is closed source, so I just ordered a couple of board for testing
![P_20220717_173558.jpg](https://user-images.githubusercontent.com/8822552/179392537-3b91ffe4-0469-4be2-8d69-9e8abfddcb1b.jpg)
